packages/react/src/fetchJsonLines.ts

```diff
@@ -6,15 +6,17 @@ export async function* fetchJsonLines<T = any>(
     requestInit?: RequestInit,
 ): AsyncGenerator<T> {
     const response = await fetch(url, requestInit);
+
     if (!response.ok) {
         throw new Error(
             `Failed to fetch: ${url} - Status: ${response.status} ${response.statusText}`,
         );
     }
 
     const reader = response.body?.getReader();
+
     if (!reader) {
-        throw new Error('No reader available for the response body.');
+        throw new Error('No reader available for the response body!');
```

Comment:
Please avoid these cosmetic changes to files. It messes up the git history. Why does this matter?

As projects mature, finding bugs gets really tricky sometimes, and change is quite often a big predictor of bugs.

It's not that these changes are likely to introduce bugs, but if you see a file that hasn't changed in 2 years, it gives you a big indication that it's going to be safe.